xarray/tests/test_interp.py

```diff
@@ -132,29 +133,60 @@ def func(obj, new_x):
     assert_allclose(actual, expected)
 
 
-@pytest.mark.parametrize("use_dask", [False, True])
-def test_interpolate_vectorize(use_dask: bool) -> None:
+@pytest.mark.parametrize(
+    "use_dask, method",
+    (
+        (False, "linear"),
+        (False, "akima"),
+        pytest.param(
+            False,
+            "makima",
+            marks=pytest.mark.skipif(not has_scipy_ge_1_13, reason="scipy too old"),
+        ),
+        pytest.param(
+            True,
+            "linear",
+            marks=pytest.mark.skipif(not has_dask, reason="dask not available"),
+        ),
+        (True, "akima"),
+    ),
+)
+def test_interpolate_vectorize(use_dask: bool, method: str) -> None:
     if not has_scipy:
         pytest.skip("scipy is not installed.")
 
-    if not has_dask and use_dask:
-        pytest.skip("dask is not installed in the environment.")
-
     # scipy interpolation for the reference
-    def func(obj, dim, new_x):
+    def func(obj, dim, new_x, method):
+        scipy_kwargs = {}
+        interpolant_options = {
+            "barycentric": "BarycentricInterpolator",
+            "krogh": "KroghInterpolator",
+            "pchip": "PchipInterpolator",
+            "akima": "Akima1DInterpolator",
+            "makima": "Akima1DInterpolator",
+        }
+
         shape = [s for i, s in enumerate(obj.shape) if i != obj.get_axis_num(dim)]
         for s in new_x.shape[::-1]:
             shape.insert(obj.get_axis_num(dim), s)
 
-        return scipy.interpolate.interp1d(
-            da[dim],
-            obj.data,
-            axis=obj.get_axis_num(dim),
-            bounds_error=False,
-            fill_value=np.nan,
-        )(new_x).reshape(shape)
+        if method in interpolant_options:
+            interpolant = getattr(scipy.interpolate, interpolant_options[method])
+            if method == "makima":
+                scipy_kwargs["method"] = method
+            return interpolant(
+                da[dim], obj.data, axis=obj.get_axis_num(dim), **scipy_kwargs
+            )(new_x).reshape(shape)
+        else:
+            scipy_kwargs["kind"] = method
+            scipy_kwargs["bounds_error"] = False
+            scipy_kwargs["fill_value"] = np.nan
+            return scipy.interpolate.interp1d(
+                da[dim], obj.data, axis=obj.get_axis_num(dim), **scipy_kwargs
+            )(new_x).reshape(shape)
```

Comment:
another nit: this reduces a little bit of indirection and makes it easier to review

```suggestion
            return scipy.interpolate.interp1d(
                da[dim], obj.data, axis=obj.get_axis_num(dim), kind=method, bounds_error=False, fill_value=np.nan
            )(new_x).reshape(shape)
```